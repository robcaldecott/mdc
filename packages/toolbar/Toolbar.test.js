import React, { Fragment } from "react";
import {
  Toolbar,
  ToolbarRow,
  ToolbarSection,
  ToolbarIcon,
  ToolbarTitle,
  ToolbarAdjust
} from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Fragment>
        <Toolbar fixed>
          <ToolbarRow>
            <ToolbarSection align="start">
              <ToolbarIcon menu>menu</ToolbarIcon>
            </ToolbarSection>
            <ToolbarSection>
              <ToolbarTitle>Title</ToolbarTitle>
            </ToolbarSection>
            <ToolbarSection align="end" shrinkToFit>
              <ToolbarIcon type="a">anchor</ToolbarIcon>
              <ToolbarIcon type="button">button</ToolbarIcon>
            </ToolbarSection>
          </ToolbarRow>
        </Toolbar>
        <ToolbarAdjust fixed>
          <div>Content</div>
        </ToolbarAdjust>
      </Fragment>
    )
  ).toMatchSnapshot();
});
