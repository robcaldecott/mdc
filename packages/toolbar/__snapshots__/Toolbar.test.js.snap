// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
Array [
  <header
    className="mdc-toolbar mdc-toolbar--fixed"
  >
    <div
      className="mdc-toolbar__row"
    >
      <section
        className="mdc-toolbar__section mdc-toolbar__section--align-start"
      >
        <span
          className="material-icons mdc-toolbar__menu-icon"
        >
          menu
        </span>
      </section>
      <section
        className="mdc-toolbar__section"
      >
        <span
          className="mdc-toolbar__title"
        >
          Title
        </span>
      </section>
      <section
        className="mdc-toolbar__section mdc-toolbar__section--align-end mdc-toolbar__section--shrink-to-fit"
      >
        <a
          className="material-icons mdc-toolbar__icon"
        >
          anchor
        </a>
        <button
          className="material-icons mdc-toolbar__icon"
        >
          button
        </button>
      </section>
    </div>
  </header>,
  <div
    className="mdc-toolbar-fixed-adjust"
  >
    Content
  </div>,
]
`;
